src/CMakeLists.txt

```diff
@@ -138,15 +138,15 @@ macro(MAKE_LIBRARY LIB_TARGET HEADER_INSTALL_DIR)
     install(
         EXPORT ff7tkTargets
         NAMESPACE ff7tk::
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/ff7tk
+        DESTINATION share/cmake/ff7tk
```

Comment:
I think you can put something like `${CMAKE_INSTALL_DATADIR}` here instead of hard coding it.

Reasoning is this one:
```
Note that some of the types' built-in defaults use the DATAROOT directory as a prefix. The DATAROOT prefix is calculated similarly to the types, with CMAKE_INSTALL_DATAROOTDIR as the variable and share as the built-in default. You cannot use DATAROOT as a TYPE parameter; please use DATA instead.
```
Source: https://cmake.org/cmake/help/latest/command/install.html#installing-directories

Also see:
```
DATAROOTDIR        read-only architecture-independent data root (share)
```
Source: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#module:GNUInstallDirs